src/services/flowTypeService.js

```diff
@@ -0,0 +1,14 @@
+const flowTypeDao = require('../models/flowTypeDao');
+const error = require('../utils/error');
+
+const getFlowTypes = async () => {
+  const flowTypes = await flowTypeDao.getFlowTypes();
+  if (flowTypes.length===0) {
+    error.throwErr(404, 'NOT_FOUND_TYPE');
+  }
+  else {
+    return flowTypes;
+  }
+}
+
```

Comment:
```suggestion
  return flowTypes;
}
```

else 없어도 될 것 같습니다 :)